src/axolotl/core/builders/causal.py

```diff
@@ -271,20 +268,34 @@ def build(self, total_num_steps):
                 self.cfg.sample_packing_eff_est
             )
 
-        if self.cfg.relora_steps:
-            training_arguments_kwargs["relora_steps"] = self.cfg.relora_steps
-            training_arguments_kwargs["relora_warmup_steps"] = (
-                self.cfg.relora_warmup_steps
-            )
-            if self.cfg.relora_anneal_steps:
+        if self.cfg.relora:
+            training_arguments_kwargs["relora_steps"] = self.cfg.jagged_restart_steps
+            if self.cfg.jagged_restart_warmup_steps:
+                training_arguments_kwargs["relora_warmup_steps"] = (
+                    self.cfg.jagged_restart_warmup_steps
+                )
+            if self.cfg.jagged_restart_anneal_steps:
                 training_arguments_kwargs["relora_anneal_steps"] = (
-                    self.cfg.relora_anneal_steps
+                    self.cfg.jagged_restart_anneal_steps
                 )
             if self.cfg.relora_prune_ratio:
                 training_arguments_kwargs["relora_prune_ratio"] = (
                     self.cfg.relora_prune_ratio
                 )
 
+        if self.cfg.jagged_restart_steps:
+            training_arguments_kwargs["jagged_restart_steps"] = (
+                self.cfg.jagged_restart_steps
+            )
+            if self.cfg.jagged_restart_warmup_steps:
+                training_arguments_kwargs["jagged_restart_warmup_steps"] = (
+                    self.cfg.jagged_restart_warmup_steps
+                )
+            if self.cfg.jagged_restart_anneal_steps:
+                training_arguments_kwargs["jagged_restart_anneal_steps"] = (
+                    self.cfg.jagged_restart_anneal_steps
+                )
```

Comment:
Confused on the difference between the two. Is jagged the new method? Why do we have both here but remove relora from input schema?